app/javascript/controllers/story_users_controller.js

```diff
@@ -0,0 +1,56 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
+  static targets = ["content_u"];
+
+  async connect() {
+    const response = await fetch(
+      `/users/${this.content_uTarget.dataset.userId}/edit`
+    );
+
+    const data = await response.json();
+
+    const stories = data.stories.map((story) => {
+      return [
+        story.id,
+        story.name.body.replace("<div>", "").replace("</div>", ""),
+        story.day,
+        story.images,
+      ];
+    });
+
+    const cls_stories = stories.map((story) => {
+      return [story[1], story[2], story[3], story[4]];
```

Comment:
```suggestion
      return [...story];
```

Reply:
hahahaha THANK U I COMPLETELY FORGOT THE SPREAD OPERATOR :)